gradle/project.gradle.kts

```diff
@@ -112,26 +114,25 @@ dependencies {
         exclude("xpp3", "xpp3_min")
     }
 
-    // Image
+    // Images
     compile("com.drewnoakes", "metadata-extractor", "2.11.0")
-    //	compile("com.twelvemonkeys.imageio", "imageio-core", "3.4.0")
-    //	compile("com.twelvemonkeys.imageio", "imageio-bmp", "3.4.0")
-    //	compile("com.twelvemonkeys.imageio", "imageio-jpeg", "3.4.0")
-    //	compile("com.twelvemonkeys.imageio", "imageio-iff", "3.4.0")
-    //	compile("com.twelvemonkeys.imageio", "imageio-icns", "3.4.0")
-    //	compile("com.twelvemonkeys.imageio", "imageio-pcx", "3.4.0")
-    //	compile("com.twelvemonkeys.imageio", "imageio-pict", "3.4.0")
-    //	compile("com.twelvemonkeys.imageio", "imageio-clippath", "3.4.0")
-    //	compile("com.twelvemonkeys.imageio", "imageio-hdr", "3.4.0")
-    //	compile("com.twelvemonkeys.imageio", "imageio-pdf", "3.4.0")
-    //	compile("com.twelvemonkeys.imageio", "imageio-pnm", "3.4.0")
-    //	compile("com.twelvemonkeys.imageio", "imageio-psd", "3.4.0")
-    //	compile("com.twelvemonkeys.imageio", "imageio-tga", "3.4.0")
-    //	compile("com.twelvemonkeys.imageio", "imageio-sgi", "3.4.0")
-    //	compile("com.twelvemonkeys.imageio", "imageio-thumbsdb", "3.4.0")
-    //	compile("com.twelvemonkeys.imageio", "imageio-tiff", "3.4.0")
-
-    compile(files(file("extra/lib").listFiles()))
+    val imageioVersion = "3.4.1"
```

Comment:
What happened to the libs in the directory? The libs were custom built I did offline because they contained rather important fixes for several bugs. It will need to be verified that this bug is indeed solved in these libraries. Unfortunately I do not quite remember what the exact problem was. Hm

Reply:
Well he released 3.4.1 by now, so there might be quite a fee fixes in the meantime. Did you create PRs for he custom fixes?

Reply:
I guess we can only hope the fixes were there. I didnt make the fixes myself, its jsut the offocial release did not contain them, but the dev branch did - the release was year behind. I just pulled the repo and build the jars. I'll hope its gonna be ok...

Reply:
Yes, if you pulled the repo it's definitely okay. The version from mavenCentral is newer than the one you built.